templates/data/postgresql/support.html

```diff
@@ -0,0 +1,731 @@
+        Canonical offers advisory services to help you design, deploy and
+        experiment with PostgreSQL solutions that fit your needs in terms of
+        security, features, high availability, scalability and observability.
+      </p>
+      <div class="p-section--shallow">
+        <div class="p-image-container--16-9 is-highlighted">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/7bf5c40d-os_infra_hardware_infra.svg",
+            alt="Canonical hardware ecosystem",
+            width="600",
+            hi_def=True,
+            loading="auto",
+            attrs={"class": "p-image-container__image"}
+            ) | safe
+          }}
+        </div>
+      </div>
+      <ul class="p-list--divided u-no-margin--bottom">
+        <li class="p-list__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            Deliver detailed design recommendations
+          </h3>
+          Our experts will work with you to understand your requirements and
+          to define the design of the PostgreSQL deployments to meet your
+          needs. Our proposed design can cover the full stack from hardware
+          dimensioning up to the optimal application settings.
+        </li>
+        <li class="p-list__item">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            Deliver Proof of Concepts
+          </h3>
+          Canonical engineers can deploy our PostgreSQL stack to the cloud or
+          data centre of your choice. Bare metal nodes, virtual machines,
+          Kubernetes and air-gapped environments are all supported.
+        </li>
+      </ul>
+      <div class="p-cta-block u-no-padding--bottom">
+        <a
+          class="p-button--positive"
+          aria-controls="data-relational-dbs-modal"
+          href="/data/postgresql#get-in-touch"
+          >Contact us</a
+        >
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>PostgreSQL training</h2>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <div class="p-image-container--16-9 is-highlighted is-cover">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/8135cf37-training.png",
+            alt="",
+            width="1800",
+            height="1015",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-image-container__image"}
+            ) | safe
+          }}
+        </div>
+      </div>
+      <p>
+        Canonical offers comprehensive training for Charmed PostgreSQL. Ramp up
+        your teams to make the most of PostgreSQL and Canonical’s automation
+        tooling, with training delivered either in-person or virtually.
+      </p>
+      <div class="p-cta-block u-no-padding--bottom">
+        <a href="https://ubuntu.com/training"
+          >Learn more about our training offers&nbsp;&rsaquo;</a
+        >
```

Comment:
please fix the closing tag here and in several places below. Looks like you're using a formatter that is not djLint

Reply:
Yes, it looks like Prettier output some things strangely. I've adjusted it, phal